README.md

```diff
@@ -1,105 +1,130 @@
+<p align = "center">
+<img src="https://media.giphy.com/media/8J8SBaqrNxbPz8jla8/giphy.gif">
+</p>
+    <details>
+      <summary>Under the Hood</summary>
+      When the user selects the apple icon, the card element's class is updated. Then, when the user clicks "My Recipes", the main section's class is updated to only display the cards with the favorited class. 
+    </details>
+   
+#### Pantry
+The user can also keep track of their pantry with this app. In the top, right corner, the user can select the "My Pantry" button to see all of the ingredients that they have in their cupboard to helo decide what recipes they want to make. 
+<p align = "center">
+<img src="https://media.giphy.com/media/6hPk6pvU7ZBH9BJFRn/giphy.gif">
+</p>
+    <details>
+      <summary>Under the Hood</summary>
+      The pantry displays from the User instance's pantry property.
+    </details>
+
+#### Recipes
+When the user selects a recipe card, the recipe expands to show the list of ingredients and instructions for making the recipe. The page will let a user know if their pantry has the necessary ingredients to cook this recipe and gives them a shopping list for the ingredients they might need. 
+<p align = "center">
+<img src="https://media.giphy.com/media/ELG5rt7fx4SgS7SkDv/giphy.gif">
+</p>
+<br>
+
+When the user has done their shopping, they can click the "Add Items to Pantry" button. If the user has all of the necessary ingredients in their pantry, they can click the "Cook Recipe" button, and those ingredients will be removed from their pantry. The user can also see when they cooked that recipe last. 
+<p align = "center">
+<img src="https://media.giphy.com/media/eSrlmC6rIqMrdoyDoz/giphy.gif">
+</p>
+    <details>
+      <summary>Under the Hood</summary>
+      When the recipe card is selected, the recipe modal is displayed and brought to the front/top, disabling all of the functionality behind/under it. The list of ingredients and instructions are displayed form the Recipe instance. The user's pantry is checked for all of the Recipe's ingredients and displays whether the user has the ingredients to make the recipe.
+    <br>
+    <br>
+    If the user has the required ingredients, they can click the "Cook Recipe" button and the User's pantry will be uodated using a a Fetch POST to remove the ingredients. If the user does not have the necessary ingredients, they can click the "Add Items to Pantry" button and the User's pantry will be updated using a Fetch POST to add the ingredients.
+    <br>
+    <br>
+    When the user decides to cook a recipe, the Recipe is added to the User's cookedRecipes property. If the user has cooked the recipe before and it already exists in that property, then the Recipe is not added, but the date is updated. 
+    </details>
+
+#### Continuous Improvement/Future Improvements
+- Check for duplicate ingredients in the API before being added to the classes
+- Style more responsively 
```

Comment:
Lets be a little more specific:
"Improve mobile responsive design so that the Recipe Type filter sidebar is shown/hidden on click."